core/src/actions/destinations.ts

```diff
@@ -175,6 +175,7 @@ export class DestinationEdit extends AuthenticatedAction {
     state: { required: false },
     options: { required: false, formatter: APIData.ensureObject },
     mapping: { required: false, formatter: APIData.ensureObject },
+    deliveryMode: { required: false },
```

Comment:
This change implies that there should be an update to the Action's test as well, confirming that you can set the deliver mode